backend/packages/identity-scan/src/scan/utils/getIdentityStorage.js

```diff
@@ -0,0 +1,48 @@
+const {
+  chain: { getApi },
+} = require("@osn/scan-common");
+const { toDecimal } = require("./unitConversion");
+
+async function getIdentityStorage(accountId) {
+  const api = await getApi();
+  let identity = {};
+  identity.accountId = accountId;
+
+  const identityInfo = await api.query.identity.identityOf(accountId);
+
+  if (!identityInfo.isSome) {
+    return identity;
+  }
+
+  const { info, judgements, deposit } = identityInfo.unwrap();
+  identity.info = {
+    display: info.display.asRaw.toUtf8(),
+    legal: info.legal.asRaw.toHuman(),
+    web: info.web.asRaw.toHuman(),
+    riot: info.riot.asRaw.toHuman(),
+    email: info.email.asRaw.toHuman(),
+    image: info.image.asRaw.toHuman(),
+    twitter: info.twitter.asRaw.toHuman(),
+    pgpFingerprint: info.pgpFingerprint.isSome
+      ? info.pgpFingerprint.unwrap().toHex()
+      : null,
+  };
+  identity.deposit = await toDecimal(deposit.toNumber());
```

Comment:
Please use [toDecimal128](https://github.com/opensquare-network/statescan-v2/blob/main/backend/packages/common/src/utils/toDecimal128.js#L4). Note we should save the planck unit value, not value with KSM/DOT as unit.